### DISCUSSION TITLE
I need your feedback!!!

### DISCUSSION THREAD

**Discussion by marshevms** [marshevms/mlfc#31] [Polls]

**Comment by volnes** [comment]
Tested on GF63-Thin-9SCSR with 
CPU: Intel i5-9300H (8) @ 4.100GHz 
GPU: Intel CoffeeLake-H GT2 
GPU: NVIDIA GeForce GTX 1650 Ti Mobile

Works good!

**Comment by Carbocode** [comment]
I use your app and it's awesome. It works very well. This + CPU Power Manager make the perfect combo as a substitute to Dragon Center.

**Comment by Petukhov18** [comment]
Thank you very much for this application.

Tested on Katana GF76 11UE with
CPU: Intel i7-11800H @ 2.30GHz
GPU: Intel TigerLake-H GT1
GPU: GeForce RTX 3060 Mobile
OS: Ubuntu 22.04.1 LTS

Sorry to bother you. Do you know how to fix this errors?
![Снимок экрана от 2022-10-10 19-18-02](https://user-images.githubusercontent.com/39758261/194913998-bba7dcc1-670a-4945-9dd8-ce07526cc6d7.png)

**Comment by Sweettooth24rus** [comment]
Tested on GL62m 7RDX with
CPU: Intel i7-7700HQ @ 2.80GHz
GPU: GeForce GTX 1050 Mobile
OS: Ubuntu 20.04.5 LTS

The temperature points were displayed on the graph, but the percentage of fan operation could not be changed, it was always 0. Because of this, the fan never worked. I had to delete the app

**Comment by volnes** [comment]
@marshevms any plans about new update?

**Comment by volnes** [comment]
@marshevms any plans about new update?
